parts/linux/cloud-init/artifacts/cgroup-memory-telemetry.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
```

Comment:
might be worth doing something like this to avoid stray issues:

```
find /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events/ -mtime +5 -type f -delete
```

this would delete events older than 5 days, which would mean we only clean up after 1440 events have been backlogged (24*60 / 5 * 5)